contracts/FleekERC721.sol

```diff
@@ -40,56 +65,36 @@ contract FleekERC721 is ERC721, FleekAccessControl {
         string gitRepository;
     }
 
-    Counters.Counter private _tokenIds;
+    /**
+     * The stored data for each AccessPoint.
+     */
+    struct AccessPoint {
+        uint256 tokenId;
+        uint256 index;
+        uint256 score;
+        string source;
+        bool contentVerified;
+        bool nameVerified;
+        address owner;
+    }
+
+    Counters.Counter private _appIds;
     mapping(uint256 => App) private _apps;
+    mapping(string => AccessPoint) private _accessPoints;
 
     /**
      * @dev Initializes the contract by setting a `name` and a `symbol` to the token collection.
      */
     constructor(string memory _name, string memory _symbol) ERC721(_name, _symbol) {}
 
     /**
-     * @dev Checks if msg.sender has the role of tokenOwner for a certain tokenId.
+     * @dev Checks if the AccessPoint exists.
      */
-    modifier requireTokenOwner(uint256 tokenId) {
```

Comment:
To double check, we don't need this for any standard right? This was part of ownable which we are not implementing I believe

Reply:
This modifier has not being used since we have implemented our own AccessControl module.